core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -51,18 +53,48 @@ angular.module('oppia').directive('imageUploader', [
             return 'This file is not recognized as an image.';
           }
 
-          if (!file.type.match('image/jpeg') &&
-            !file.type.match('image/gif') &&
-            !file.type.match('image/jpg') &&
-            !file.type.match('image/png') &&
-            !file.type.match('image/svg\\+xml')) {
+          var imageTypeMapping = {
+            jpeg: {
+              format: 'image/jpeg',
+              condition: [/jp(e?)g$/, /\.jp(e?)g$/],
```

Comment:
Here and below, I think you can simplify these into a single regex? Basically the "." at the start is optional.

Reply:
These regex are for two different conditions for filetype and extension.

Reply:
OK, but my point still stands, at least for this line. You should format it like the others.

Reply:
I have formatted this like the others but I am not sure if this is what you had expected.